luma/core/virtual.py

```diff
@@ -227,8 +229,14 @@ def println(self, text=""):
         Prints the supplied text to the device, scrolling where necessary.
         The text is always followed by a newline.
         """
-        self.puts(text)
-        self.newline()
+        if self.word_wrap:
+            text = wrap(text, width=self.width)
```

Comment:
Hello world is missing because it has scrolled off the top. Clearly the word wrap is counting the ANSI colour code characters rather than interpreting them as zero width.

Wonder whether we need a home grown text wrapper that does take ANSI color codes into account?

Maybe just leave as-is for now.

Interestingly, it has exposed another bug (which was there before your commits) when scrolling when the background colour is not black - see the yellow bar at the bottom.